packages/manager/modules/sms/src/sms/compose/telecom-sms-sms-compose.controller.js

```diff
@@ -15,6 +15,7 @@ import sum from 'lodash/sum';
 import sumBy from 'lodash/sumBy';
 import union from 'lodash/union';
 import moment from 'moment';
+import _ from 'lodash';
```

Comment:
Prefer using `import fooMethod from 'lodash/fooMethod';`.

Reply:
fooMethod doesn't exist in the latest version of lodash (4.17.11).

Reply:
I mean you can import only needed methods (refer to L1-L7)